hd/etc/updind.txt

```diff
@@ -914,6 +914,10 @@ function changeCalendar(e,v,m,c) {
     [*modify::person/persons]0[:] %first_name;.%if;(occ = "")0%else;%occ;%end; %surname;
   %else;
     [*add::person/persons]0%end;
+  <input type="hidden" name="old_fn"  value="%first_name;">
+  <input type="hidden" name="old_sn"  value="%surname;">
```

Comment:
It's better to use %first_name_key; and %surname_key; to avoid %20 for spaces in URL.